tools/onnx_subgraph/onnx.proto

```diff
@@ -0,0 +1,871 @@
+//
+// WARNING: This file is automatically generated!  Please edit onnx.in.proto.
+//
```

Comment:
1/ it would be better to generate at configure or compile time instead from onnx.in.proto instead of adding this generated file in source tree.

2/ I don't see `onnx.in.proto` file in this PR. how are you going to mange change for this file?

Reply:
I see...